### ISSUE TITLE
Previewing an Item during the Workflow approval process not possible in Angular UI

### ISSUE THREAD

**Issue by heathergreerklein** [DSpace/dspace-angular#772]
This is a duplicate of [DS-4490](https://jira.lyrasis.org/browse/DS-4490)

Estimate from release plan: none
Expressing interest: none

Pulled out from a larger ticket https://jira.lyrasis.org/browse/DS-4311

Previewing an Item should be possible (via a button or similar alongside the "Approve" / "Reject" / "Edit" buttons). This is a low priority task, but was possible in XMLUI. It's a minor usability issue.

**Comment by tdonohue** [comment]
In today's meeting, this task came up as a possible quick-fix for #721 (if that turns out to be harder than expected).  This ticket was estimated at *26 hours*.

**Comment by tdonohue** [comment]
Simple solution being implemented for 7.0, see #1236.  This ticket is about enhancing that solution as needed, or possibly a popup modal style preview of an Item.

**Comment by benbosman** [comment]
Some details about the issue of the preview I'm still seeing on the demo site:
https://demo7.dspace.org/mydspace?configuration=workspace&f.namedresourcetype=workspace,authority&spc.page=1&query=dspace.entity.type:Person&spc.rpp=1&view=detailed
This is displayed as
<img width="581" alt="Schermafbeelding 2022-02-10 om 16 40 11" src="https://user-images.githubusercontent.com/1528033/153442396-40ce5f2d-a2c8-41ea-9463-ce48960e4a2a.png">

This doesn't match https://demo7.dspace.org/entities/person/44af20c7-5eec-4e47-9d8b-1f08fbed2b83, most likely because it's not using the item display component yet

**Comment by tdonohue** [comment]
@benbosman : That "doesn't match" link you shared doesn't work for me yet (it's possible because it's an in-progress submission, I just don't have access to see it).   

But, I did want to note that I don't believe it can always be an "exact match" for the Item page. Keep in mind that in the Workflow screen, this "Preview" is embedded in a paginated search results listing...so, we may be limited in what components can be displayed in that view... and I would expect that trying to match all the features of the Item page, especially any related to Entities (like showing relationships, etc) might be very complex to achieve in this display.

That said, I do agree with you that, where possible, it'd be nice to share components between the Item page and this Preview page.  But, I don't see that as a requirement for this feature.  

The feature here was just to provide a "Preview" display that works in the same way that in the DSpace 6 XMLUI.  In 6.x XMLUI, the Preview did not show you the exact Item page, but rather an example of how it'd look, including being able to see the metadata & view/download files.  This at least gave you a general idea/preview of what this Item will look like before you review it.

So, in my opinion, the screenshot you show above might be "good enough" for this preview feature.  It shows the Item's metadata in a manner similar to the Item page, and it *should show* the files (if any are uploaded).  That's very similar to how this worked in DSpace 6 XMLUI.

In other words, I'd rather we find a "good enough", simple approach here, instead of trying to make this look exactly the same as the Item page...as I think our limited budget would be better spent building other new features.  We can always revisit this "Preview" page later on if/when we feel the need to enhance the display further.
